### PR TITLE
Rule isArray - add support for custom message

### PR DESCRIPTION
**Pull request by edcapetti-zz**
Added support for custom message.
#15

### CODE REVIEW

File: src/Rule/IsArray.php [hunk 1]
```diff
@@ -10,6 +10,19 @@
...
+    /**
+     * @var string | null
+     */
+    private $message;
+
+    /**
+     * IsArray constructor.
+     * @param string|null $message
+     */
+    public function __construct(string $message = null)
```
Comment (jwillp) [inline src/Rule/IsArray.php]: Please add make the type nullable: 
```php
?string $message = null
```

File: src/Rule/IsIPAddress.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<?php
```
Comment (jwillp) [inline src/Rule/IsIPAddress.php]: This should not be part of this commit.

File: tests/Rule/IsIPAddressTest.php [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<?php
```
Comment (jwillp) [inline tests/Rule/IsIPAddressTest.php]: This should not be part of this commit.

**Review by jwillp — CHANGES_REQUESTED** [review]
Hi @edcaracas, thank you for your contribution.
There's only one small detail to take care of.
Once you do this, I will approve this PR and merge it. :)

File: src/Rule/IsArray.php (hunk 1 above)
Comment (edcapetti-zz) [reply src/Rule/IsArray.php]: Fixed.
https://github.com/Morebec/Validator/pull/16/files/c32f74bf3b00242ac0c7a3f5b96800c1e49063e6..b4a69c683baec88f7afc0ba526d6f48ea801a5f0

**Review by jwillp — APPROVED** [review]
Thank you!
